include/nbl/builtin/hlsl/fft/common.hlsl

```diff
@@ -0,0 +1,44 @@
+#ifndef _NBL_BUILTIN_HLSL_FFT_COMMON_INCLUDED_
+#define _NBL_BUILTIN_HLSL_FFT_COMMON_INCLUDED_
+
+#include "nbl/builtin/hlsl/complex.hlsl"
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+#include "nbl/builtin/hlsl/fft/common/twiddles.hlsl"
+
+namespace nbl 
+{
+namespace hlsl
+{
+namespace fft::common {
```

Comment:
1. HLSL doesn't support nested namespace syntax
2. use `detail` or `impl` for things you don't want to make public to others